src/pages/home/report/ReportActionCompose.js

```diff
@@ -372,6 +372,17 @@ function ReportActionCompose({
         focusWithDelay(textInputRef.current)(shouldDelay);
     }, []);
 
+    const willOpenNextModal = useRef(false);
+    const isKeyboardVisibleWhenShowingModal = useRef(false);
```

Comment:
same here

Reply:
I meant something like `isNextModalWillOpenRef`, `isKeyboardVisibleWhenShowingModalRef` because these are refs, not just boolean